docs/gui.rst

```diff
@@ -177,14 +177,19 @@ Every event have a key and type.
   ...
   ti.GUI.LMB     # Left Mouse Button
   ti.GUI.RMB     # Right Mouse Button
+  ti.GUI.WHEELUP    # Mouse wheel scroll up
+  ti.GUI.WHEELDOWN  # Mouse wheel scroll down
+  ti.GUI.WHEELLEFT  # Wheel scroll to left
+  ti.GUI.WHEELRIGHT # Wheel scroll to right
```

Comment:
Having four sounds too complex for me, let's just use ti.GUI.WHEEL and use delta to distinguish scroll direction.